Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM alpine
+
+RUN apk update
+RUN apk add git openssh-client curl jq
+COPY update-version-wrapper /
+CMD /update-version-wrapper
```

Comment:
It'd be nice to have this running under a specific user (see https://github.com/GPII/universal/blob/master/Dockerfile for an example, although here the 'node' user doesn't exist obviously -- "nobody" is a good candidate)

Reply:
`nobody` doesn't seem like a good choice since it has no login shell and its home dir is /, where it has no permissions to do anything.

I created a dedicated user `app`.

Reply:
This turned out to be a bit tricky. The new `app` user in the container can't access the volume that is passed in with the ssh creds. I couldn't make it work even with the files owned by the same UID in the container and on the host.

I guess I need to use data volumes or adopt one of the other strategies here: https://stackoverflow.com/questions/23544282/what-is-the-best-way-to-manage-permissions-for-docker-shared-volumes

I'm still working on this.

Reply:
It works now without any further changes. I think what happened was:
* I must have missed a step getting my local changes over to i46 (forgetting to push to Docker Hub seems the most likely culprit).
* I forgot the selinux `:Z` flag on the volume in my manual testing on the command line (although there is nothing in the audit.log). The ansible code has `:Z` and so the full pipeline was working even.

Anyway, the version of the image that drops root privileges is running successfully on i46.